### PR TITLE
Categories: handle symlinks in the build system

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

**Review by tintou — CHANGES_REQUESTED** [review]
```
Tried to install symlink to missing file /usr/local/share/icons/elementary/apps/16/applications-internet.svg
```

**Comment by danirabbit** [comment]
@tintou sorry about that :sweat_smile: fixed

File: categories/meson.build [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    # the second is the link
+    ['applications-internet', 'preferences-desktop-online-accounts'],
+    ['multimedia-volume-control', 'preferences-desktop-sound'],
+    ['preferences-desktop-locale', 'config-language'], # gnome language selector
+    ['preferences-system-power', 'gnome-power-manager']
+]
+
+foreach link : links
+    foreach size: sizes
+        if (fs.exists(link[1] + '.svg'))
```
Comment (tintou) [inline categories/meson.build:14]:
```suggestion
        if (fs.exists(link[1] + '.svg'))
```
This looks wrong to me, is anything installed?

**Comment by danirabbit** [comment]
@tintou Okay sorry for so many pings :sweat_smile: I think it's actually fixed now!

**Pull request merged**